### PR TITLE
Fix not git repository fatal

### PR DESCRIPTION
**Pull request by jhadvig**
@stevekuznetsov I've added the revert to your revert commit, even tough I'm not really sure why we need it since your revert has not been merged so there are 3 commints in my PR. Also added the stege statuses to the status config file.
PTAL 
fixes https://github.com/openshift/origin/issues/13806

### CODE REVIEW

**Comment by stevekuznetsov** [comment]
@jhadvig pushed the revert, sorry -- rebase please

File: config/test_cases/test_pull_request_openshift_ansible_extended_conformance_install_with_status_check.yml [hunk 1]
```diff
@@ -7,12 +7,10 @@ overrides:
...
 extensions:
-  sync_repos:
-    - name: "aos-cd-jobs"
   actions:
     - type: "host_script"
       title: "check other job statuses"
-      repository: "aos-cd-jobs"
       script: |-
-        git checkout sjb
+        wget https://raw.githubusercontent.com/openshift/aos-cd-jobs/sjb/check-pull-request-status.py
```
Comment (stevekuznetsov) [inline config/test_cases/test_pull_request_openshift_ansible_extended_conformance_install_with_status_check.yml]: I'd rather use `scp` to grab them from the VM -- we know the SHA from which that repo was cloned there, etc -- this is a little less inspectable.

**Comment by jhadvig** [comment]
@stevekuznetsov comment addressed && rebased

File: config/test_cases/test_pull_request_openshift_ansible_extended_conformance_install_with_status_check.yml [hunk 2]
```diff
@@ -7,12 +7,10 @@ overrides:
...
     - name: "origin"
 extensions:
-  sync_repos:
-    - name: "aos-cd-jobs"
   actions:
     - type: "host_script"
       title: "check other job statuses"
-      repository: "aos-cd-jobs"
       script: |-
-        git checkout sjb
```
Comment (stevekuznetsov) [inline config/test_cases/test_pull_request_openshift_ansible_extended_conformance_install_with_status_check.yml]: You'll still need a step on the remote host to do this checkout

```yaml
 - type: "script"
   title: "check out correct aos-cd-jobs branch"
   repository: "aos-cd-jobs"
   script: |-
     git checkout sjb
```

File: check-pull-request-status.py [hunk 3]
```diff
@@ -23,13 +25,14 @@
...
-    'aos-ci-jenkins/OS_3.5_containerized',
-    'aos-ci-jenkins/OS_3.5_containerized_e2e_tests',
-    'continuous-integration/travis-ci/pr'
-]
+blocking_statuses = []
+
+with open(status_config_file) as status_config:
+    config = load(status_config)
+    if target_branch not in config:
+        print("[ERROR] No GitHub PR statuses have been configured for branch '{}'.".format(target_branch))
```
Comment (stevekuznetsov) [inline check-pull-request-status.py]: I swear we had an `exit(1)` in  here before ...

Comment (jhadvig) [inline check-pull-request-status.py]: no idea how it disappeared

**Pull request merged**